desk/app/reel.hoon

```diff
@@ -174,18 +188,20 @@
       %reel-want-token-link
     =+  !<(=token:reel vase)
     :_  this
+    =/  full-token
+      ?^  (rush token flag)  token
+      (rap 3 (scot %p our.bowl) '/' token ~)
     =/  result=(unit [cord cord])
-      ?.  (~(has by our-metadata) token)  `[token '']
-      `[token (url-for-token vic token)]
+      ?.  (~(has by our-metadata) full-token)  `[full-token '']
+      `[full-token (url-for-token vic full-token)]
     ~[[%pass [%token-link-want token ~] %agent [src dap]:bowl %poke %reel-give-token-link !>(result)]]
       %reel-give-token-link
     =+  !<(result=(unit [cord cord]) vase)
     ?~  result  `this
     :_  this
     =/  [token=cord url=cord]  u.result
-    ?:  =('' url)
-      ~[[%give %fact ~[[%token-link (scot %p src.bowl) token ~]] %json !>(~)]]
-    ~[[%give %fact ~[[%token-link (scot %p src.bowl) token ~]] %json !>(s+url)]]
+    =/  path  (stab (cat 3 '/token-link/' token))
```

Comment:
Some smell on this not being `/token-link/(scot %t token)`, but iirc that's for backcompat reasons that are out of scope here, right?

Reply:
(Also some of the `cord`s in the surrounding code should probably be `token:reel`s.)

Reply:
nvm we can't do this